styles/light/editTags.mcss

```diff
@@ -0,0 +1,13 @@
+StagedTags {
+  margin: 1rem 0 1rem 1rem
+  display: inline-block
+}
+
+EditTags {
+  margin: 1rem 1rem 1rem 0
+  display: inline-block
+  input.tags {
+    font-size: 1rem
+    padding: .4rem
+  }
+}
```

Comment:
Same as https://github.com/ssbc/patchwork/pull/748#discussion_r216378274, this also looks like a duplicate file.